src/main/java/org/jabref/gui/undo/UndoableInsertString.java

```diff
@@ -15,14 +14,10 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(UndoableInsertString.class);
 
     private final BibDatabase base;
-    private final BasePanel panel;
     private final BibtexString string;
 
-
-    public UndoableInsertString(BasePanel panel, BibDatabase base,
-            BibtexString string) {
+    public UndoableInsertString(BibDatabase base, BibtexString string) {
```

Comment:
also move to the `string` package?

Reply:
I was thinking about this, but decided against it, as in the undo package all other undo methods are also (UndoableRemoveEntry etc)